tests/ExpressionTest.php

```diff
@@ -164,7 +164,7 @@ public function testConstructor4()
         $this->assertSame(
             'testing "hello, world"',
             $this->e(
-                'testing "[]"',
+                'testing []',
```

Comment:
@georgehristov @DarkSide666 how would like you to see this fixed? - remove the test completely?